### PR TITLE
turn definitions into definition lists

### PR DESCRIPTION
**Pull request by robUx4**
Matching the RFC Editor format

### CODE REVIEW

File: index_matroska.md [hunk 1]
```diff
@@ -118,11 +118,14 @@ when, and only when, they appear in all capitals, as shown here.
...
+`Matroska`:
+: A multimedia container format based on EBML (Extensible Binary Meta Language).
 
-`Matroska Reader`: A `Matroska Reader` is a data parser that interprets the semantics of a Matroska document and creates a way for programs to use `Matroska`.
+`Matroska Reader`:
+: A data parser that interprets the semantics of a Matroska document and creates a way for programs to use `Matroska`.
 
-`Matroska Player`: A `Matroska Player` is a `Matroska Reader` with a primary purpose of playing audiovisual files, including `Matroska` documents.
+`Matroska Player`:
+: A `Matroska Reader` with a primary purpose of playing audiovisual files, including `Matroska` documents.
```
Comment (retokromer) [inline index_matroska.md:128]: This sounds strange to me: a Matroska Reader… including Matroska documents. Is it not the primary purpose to play Matroska files and others are the “nice to have” part?

Comment (robUx4) [reply index_matroska.md:128]: Yes, it's a bit odd. You may raise a separate issue as this patch is a formatting patch (although it changes the text a bit to make it more logical with the formatting).

Comment (robUx4) [reply index_matroska.md:128]: The text reduction is similar to [this EBML patch](https://github.com/cellar-wg/ebml-specification/commit/06545b4e57e51db085370e2e5f1ee3c7fdde0e28).

Comment (retokromer) [reply index_matroska.md:128]: Yep, I’ll. I didn’t notice it in the old version.

**Pull request merged**